File: Napakalaki/src/NapakalakiGame/Cultist.java
/**
 * Cultist.java
 */

package NapakalakiGame;

/**
 * Representa una carta de sectario.
 */
public class Cultist {
    
    private String name;
    private int gainedLevels;
    
    public Cultist(String name, int gainedLevels) {
        this.name = name;
        this.gainedLevels = gainedLevels;
    }
    
    public int getGainedLevels() {
        return gainedLevels;
    }
    
    @Override
    public String toString() {
        return name + ", niveles ganados: " + Integer.toString(gainedLevels);
    }
    
}

File: Napakalaki/lib/cultist.rb
# encoding: utf-8

module NapakalakiGame
  
  class Cultist
    def initialize(name, gainedLevels)
      @name = name
      @gainedLevels = gainedLevels # Niveles adicionales que obtiene el jugador
    end

    def getGainedLevels
      @gainedLevels
    end
    
    def to_s
      @name + ", niveles ganados: #{@gainedLevels}"
    end

  end # Cultist

end # NapakalakiGame